src/core/toga/widgets/base.py

```diff
@@ -66,7 +66,7 @@ def add(self, child):
 
         child.app = self.app
         if self._impl:
-            self._impl.add_child(child._impl)
+            self._impl.add_canvas_to_child(child._impl)
```

Comment:
Is this a stray search and replace? It doesn't seem right...

Reply:
Yes, I'm not sure how I did that, thanks for catching my error!